jumpscale/install/codeserver-install.py

```diff
@@ -1,6 +1,8 @@
 from jumpscale.core.base import StoredFactory
 from jumpscale.servers.threebot.threebot import ThreebotServer
+from jumpscale.god import j
 
 factory = StoredFactory(ThreebotServer)
 threebot = factory.get("default")
-threebot.packages.add("/sandbox/code/github/js-next/js-sdk/jumpscale/packages/codeserver/")
+threebot.packages.add(j.packages.codeserver.__file__.rsplit("/", 1)[0])
```

Comment:
why not using j.sal.fs facilities to get accurate paths instead of rsplit, 1, [0] .. .etc?

Reply:
done